Sources/VSM/Documentation.docc/ComprehensiveGuide/ModelDefinition.md

```diff
@@ -8,7 +8,7 @@ One of the unique things about the VSM architecture is its philosophy on how mod
 
 > Tip: Building VSM models requires an understanding of [finite state machines](https://en.wikipedia.org/wiki/Finite-state_machine) and simple [recursion](https://www.vadimbulavin.com/recursion-in-swift/).
 
-There are 4 slightly different, yet equally viable patterns for building models in VSM. You can read more about each pattern in <doc:ModelPatterns>. In this article, we will be focusing on the "Builder" pattern where each model implements its functionality via type extensions.
+There are multiple styles for building models in VSM. You can read more about each pattern in <doc:ModelPatterns>. In this article, we will be focusing on the "Protocol" pattern where each model implements its functionality by creating structs that implement the protocol requirements. We recommend that you use the "Protocol" model style by default.
```

Comment:
? - Do we want to include any reasoning at this point for why we recommend the Protocol style?

Reply:
I have a follow-up PR that is queued up for approval after this one which goes into depth about this pattern and its benefits in the linked <doc:ModelPatterns> article. I don't want to side-rail too much about it directly in the model definition article.